src/data/complex/exponential.lean

```diff
@@ -503,11 +503,22 @@ mul_div_cancel' _ two_ne_zero'
 lemma two_cosh : 2 * cosh x = exp x + exp (-x) :=
 mul_div_cancel' _ two_ne_zero'
 
+/-- The real definition of `sinh`-/
+lemma sinh_def (x : ℝ) : sinh x = (exp x - exp (-x)) / 2 :=
+by simp only [sinh, complex.sinh, complex.div_re, complex.exp_of_real_re, complex.one_re,
+  bit0_zero, add_zero, complex.sub_re, euclidean_domain.zero_div, complex.bit0_re,
+  complex.one_im, complex.bit0_im, mul_zero, ← complex.of_real_neg, complex.norm_sq];
+  ring
```

Comment:
```suggestion
by { simp only [sinh, complex.sinh, complex.div_re, complex.exp_of_real_re, complex.one_re,
  bit0_zero, add_zero, complex.sub_re, euclidean_domain.zero_div, complex.bit0_re,
  complex.one_im, complex.bit0_im, mul_zero, ← complex.of_real_neg, complex.norm_sq],
  ring }
```
(unless the `simp` creates multiple goals. I haven't checked)

Reply:
Done!